velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,153 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, wilsonInterval) {
```

Comment:
Separate this test into individual ones for wilson_interval_lower and wilson_interval_upper since this one has got very long.

Reply:
done